test/test_models_detection.py

```diff
@@ -32,7 +32,7 @@ def test_detpreprocessor(mock_pdf):  # noqa: F811
     assert batched_docs[-1].shape[0] == (8 * num_docs) % batch_size
 
     # Repr
-    assert repr(processor) == 'DetectionPreProcessor(output_size=(512, 512), mean=[0.5 0.5 0.5], std=[1. 1. 1.])'
+    assert repr(processor) == 'DetectionPreProcessor(output_size=(512, 512), mean=[0.798, 0.785, 0.772], std=[0.264, 0.274, 0.287])'
```

Comment:
The repr does not include the commas between figures ;)

Reply:
And it's too long for flake8

Reply:
yes I changed it